helm/superset/Chart.yaml

```diff
@@ -32,10 +32,10 @@ maintainers:
 version: 0.7.7
```

Comment:
Need to update the chart version

Reply:
Addressed: Updated chart version from 0.7.7 to 0.7.8